rules/aip0132/request_show_deleted_required_test.go

```diff
@@ -36,14 +36,14 @@ func TestRequestShowDeletedRequired(t *testing.T) {
 				message {{.Method}}Request {}
 			`, test)
 			problems := requestShowDeletedRequired.Lint(f)
-			if diff := test.problems.SetDescriptor(f.GetMessageTypes()[1]).Diff(problems); diff != "" {
+			if diff := test.problems.SetDescriptor(f.Messages().Get(1)).Diff(problems); diff != "" {
 				t.Error(diff)
 			}
 		})
 	}
 }
 
-// Regression test for https://github.com/googleapis/api-linter/issues/854.
+// Regression test for https://github.com/googleapis/api-linter/v2/issues/854.
```

Comment:
```suggestion
// Regression test for https://github.com/googleapis/api-linter/issues/854.
```
Valid Issue link caught in refactoring :)